src/SUMMARY.md

```diff
@@ -30,6 +30,7 @@
       - [AppArmor](./config/security/apparmor.md)
    - [Date and Time](./config/date-time.md)
    - [Kernel](./config/kernel.md)
+   - [Power Management](./config/power-management.md)
```

Comment:
This should become "Power", but then it's hardly descriptive. Is "Power management and saving" too big/bad?

Reply:
i guess i regard "power saving" as an aspect of "power management". Thinking about it some more, my preferred structure for `power-management.md` would be more like:

    # Power Management
    ## acpid
    ## elogind
    ## Power Saving - tlp

WDYT?

Reply:
That works, I think.

Reply:
If we decide to document more power saving stuff, they can get sublevel headers later.

Reply:
Thanks, PR updated.